Description: change to a bigger resolution, remove fullscreen
 the app was made at a time when screens were not so big
 and the app start only in fullscreen
 that could be annoying, so we start in windowed mode
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/linux/Config.txt
+++ b/linux/Config.txt
@@ -1,4 +1,4 @@
-ViewportWidth = 1920;
-ViewportHeight = 1080;
+ViewportWidth = 1280;
+ViewportHeight = 720;
 MSAA = 4;
-Fullscreen = True;
+Fullscreen = False;
